hardware_interface/include/hardware_interface/actuator_interface.hpp

```diff
@@ -22,7 +22,10 @@
 #include "hardware_interface/handle.hpp"
 #include "hardware_interface/hardware_info.hpp"
 #include "hardware_interface/types/hardware_interface_return_values.hpp"
-#include "hardware_interface/types/hardware_interface_status_values.hpp"
+#include "rclcpp_lifecycle/node_interfaces/lifecycle_node_interface.hpp"
```

Comment:
why not inheriting from the `lifecycle_node_interface`? That way we have all the function signatures for free (and their default implementation as well if wanted).